src/rgw/rgw_common.cc

```diff
@@ -1362,7 +1362,7 @@ bool verify_bucket_permission(const DoutPrefixProvider* dpp,
 			      const boost::optional<Policy>& bucket_policy,
                               const vector<Policy>& identity_policies,
                               const vector<Policy>& session_policies,
-                              const uint64_t op)
+                              const uint64_t op, bool* acl_required)
```

Comment:
can we find a better name for this? i would interpret this to control whether authorization should require an acl

maybe something like `acl_was_required` or `granted_by_acl`?